### ISSUE TITLE
 System.ArgumentOutOfRangeException BusinessCentral.LinterCop.Design.Rule0021BuiltInMethodImplementThroughCodeuni

### ISSUE THREAD

**Issue by pri-kise** [StefanMaron/BusinessCentral.LinterCop#343]
I receive the follwoing warnings attached to my app.json file.
![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/56249171/aad1bd4f-88bc-4a7b-921f-bb672092eadd)

```
Analyzer 'BusinessCentral.LinterCop.Design.Rule0021BuiltInMethodImplementThroughCodeunit' threw an exception of type 'System.ArgumentOutOfRangeException' with message 'System.ArgumentOutOfRangeException: Index and length must refer to a location within the string. (Parameter 'length')
   at System.String.Substring(Int32 startIndex, Int32 length)
   at BusinessCentral.LinterCop.Design.Rule0021BuiltInMethodImplementThroughCodeunit.CheckBuiltInMethod(OperationAnalysisContext ctx)
   at Microsoft.Dynamics.Nav.CodeAnalysis.Diagnostics.AnalyzerExecutor.<>c__DisplayClass53_1.<ExecuteOperationAction>b__1() in D:\a\_work\1\s\source\Prod\Microsoft.Dynamics.Nav.CodeAnalysis\DiagnosticAnalyzer\AnalyzerExecutor.cs:line 766
   at Microsoft.Dynamics.Nav.CodeAnalysis.Diagnostics.AnalyzerExecutor.ExecuteAndCatchIfThrows_NoLock(DiagnosticAnalyzer analyzer, Action analyze, Nullable`1 info) in D:\a\_work\1\s\source\Prod\Microsoft.Dynamics.Nav.CodeAnalysis\DiagnosticAnalyzer\AnalyzerExecutor.cs:line 1102'
```

**Comment by Arthurvdv** [comment]
Could it be you're running an "old" version of the LinterCop.dll? This issue should be resolved with the [v0.30.5](https://github.com/StefanMaron/BusinessCentral.LinterCop/releases/tag/v0.30.5) release.

Maybe run the `LC: Download Linter Cop` command in VS Code without any project/workspace open to retrieve te latest version.

**Comment by pri-kise** [comment]
I had close all VS Code Windows. 
Now it's working again.
